app/src/features/apiClient/screens/environment/components/VariablesList/VariablesList.tsx

```diff
@@ -139,8 +151,16 @@ export const VariablesList: React.FC<VariablesListProps> = ({
     recordsCount: dataSource.length,
   });
 
+  const hasDataSourceUpdated = useRef(false);
+
   useEffect(() => {
+    if (hasDataSourceUpdated.current) {
+      return;
+    }
+
     if (variables) {
+      hasDataSourceUpdated.current = true;
+
```

Comment:
This will prevent the initial values of the env variables to sync when someone else in the same workspace updates the table